src/app/mail/mail-trash-list/mail-trash-list.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, OnInit } from '@angular/core';
+// Models
+import { Mail } from '../../store/models';
+// Rxjs
+import { Observable } from 'rxjs/Observable';
+// Store
+import { Store } from '@ngrx/store';
+import { getMails } from '../../store/selectors';
+import { DeleteMail, GetMails } from '../../store/actions';
+import { OpenPgpService } from '../../store/services/openpgp.service';
+import { OnDestroy, TakeUntilDestroy } from 'ngx-take-until-destroy';
+
+@TakeUntilDestroy()
+@Component({
+  selector: 'app-mail-trash-list',
+  templateUrl: './mail-trash-list.component.html',
+  styleUrls: ['./mail-trash-list.component.scss']
+})
+export class MailTrashListComponent implements OnInit, OnDestroy {
+  mails: Mail[];
+
+  getMailsState$: Observable<any>;
+  readonly destroyed$: Observable<boolean>;
+
+  // Public property of boolean type set false by default
+  public isComposeVisible: boolean = false;
```

Comment:
blind copy paste, why?
what is the purpose of this variable here : `isComposeVisible `